utils/windowfunction/idborderby.cpp

```diff
@@ -172,8 +172,11 @@ int WideDecimalCompare::operator()(IdbCompare* l, Row::Pointer r1, Row::Pointer
     l->row2().setData(r2);
 
     int ret = 0;
-    int128_t v1 = *(l->row1().getBinaryField_offset<int128_t>(keyColumnOffset));
-    int128_t v2 = *(l->row2().getBinaryField_offset<int128_t>(keyColumnOffset));
+
+    int128_t v1, v2;
+    l->row1().getInt128Field(fSpec.fIndex, v1);
```

Comment:
I would ask you to solve issue around wrong keyColumnOffset if it is possible b/c it allows us to skip needless dereferencing operation getting the values.

Reply:
I haven't yet addressed this part yet. I have added a comment to MCOL-4188 for this so we don't forget.

Reply:
OK. Go ahead. I'll handle this.